### PR TITLE
Test enhancement

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log
- Remove `phpunit` execution file because it's useless.
- Add `^7.2` version inside `require` block in `composer.json`.

### CODE REVIEW

**Comment by rap2hpoutre** [comment]
@peter279k Thank you for your contribution!

> Remove phpunit execution file because it's useless.

Woops. You are right!

> Add ^7.2 version inside require block in composer.json.

I prefer : `^7.0` (which is still maintained). Could you edit your PR?

**Comment by peter279k** [comment]
@rap2hpoutre, thank you for your reply. I appreciate your review.

I will add the `^7.0` and add `7.0` and `7.1` tests in Travis CI build :).

File: composer.json [hunk 1]
```diff
@@ -10,8 +10,11 @@
         }
     ],
     "minimum-stability": "stable",
+    "require": {
+        "php": "^7.0"
+    },
     "require-dev": {
-        "phpunit/phpunit": "^7.0.2"
+        "phpunit/phpunit": "^6.5 || ^7.0.2"
```
Comment (peter279k) [inline composer.json:17]: The PHPUnit version `7.0+` only supports `php-7.1+`.

To be compatible with PHP version `7.0`, we need to add the `^6.5` additionally inside `require-dev` block.

**Comment by rap2hpoutre** [comment]
Thank you so much for your contribution!

File: composer.json (hunk 1 above)
Comment (rap2hpoutre) [reply composer.json:17]: +1 :)

**Pull request merged**